_layouts/act_implementation.html

```diff
@@ -0,0 +1,246 @@
+---
+layout: standalone_resource
+type_of_guidance: ""
+---
+
+{%- assign implementations = site.data.wcag-act-rules.implementations %}
+{%- assign requirements = site.data.wcag-act-rules.requirements %}
+{%- assign implementation = implementations[page.implementation_key] %}
+{%- assign implementationInfo = site.data.wcag-act-rules.act-implementations | where: "uniqueKey", page.implementation_key | first %}
+
+<section class="doc-note-box act-sticky">
+  <p>This page is <strong>under development</strong> and has not been approved by the working group.</p>
+</section>
+
+<aside class="box"> 
+  <header class="box-h box-h-icon"> 
+    Implementation Summary 
+  </header>
+  <div class="box-i"><p>
+    {% if implementationInfo.homepage -%}
+      <a href="{{ implementationInfo.homepage }}">
+        {{- implementationInfo.name -}}
+      </a>
+    {%- else %}
+      {{ implementationInfo.name }}
+    {% endif %}
+    {% unless implementation.versionDate %}
+      version {{ implementation.version }} 
+    {% endunless %}
+    is an {{ implementationInfo.type }}
+    {% if implementationInfo.vendorHomepage %}
+      by <a href="{{ implementationInfo.vendorHomepage }}">
+        {{- implementationInfo.vendor -}}
+      </a>.
+    {% else %}
+      by {{ implementation.vendor }}.
+    {% endif %}
+
+    {% if implementation.versionDate %}
+      Version {{ implementation.version }} 
+      was published on {{ implementation.versionDate | date: '%e %B %Y' }}.
+    {% endif %}
+
+    This ACT implementation tests for
+    {% for standard in implementationInfo.standards %}
+      {%- if forloop.first == true -%}
+        {{ standard }}
+      {%- elsif forloop.last == false -%}
+        , {{ standard }}
+      {%- else -%}
+        , and {{ standard }}
+      {%- endif -%}
+    {%- endfor %}.
+    See <a href="../#understandingactconsistency">understanding ACT consistency</a>.
+  </p></div>
+</aside>
+
+<table class="act-rules-summary">
+  <caption>Number of Rules Implemented</caption>
+  <tr>
+    <th></th>
+    <th>Consistent</th>
+    <th>Partially consistent</th>
+  </tr>
+  <tr>
+    <th>WCAG 2 rules</th>
+    <td>{{ implementation.approvedRules.complete }}</td>
+    <td>{{ implementation.approvedRules.partial + implementation.approvedRules.minimal }}</td>
```

Comment:
You can't use expressions here in liquid templates. 
Need to assign a variable first. 
Is throwing build errors

Reply:
It's not erroring, but it's only showing the first number. Still wrong.